instrumentation/jmx-metrics/javaagent/src/test/java/io/opentelemetry/instrumentation/javaagent/jmx/JmxMetricInsightInstallerTest.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.javaagent.jmx;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.opentelemetry.instrumentation.jmx.yaml.JmxConfig;
+import io.opentelemetry.instrumentation.jmx.yaml.JmxRule;
+import io.opentelemetry.instrumentation.jmx.yaml.RuleParser;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class JmxMetricInsightInstallerTest {
+  public static final String PATH_TO_ALL_EXISTING_RULES = "src/main/resources/jmx/rules";
+  private static RuleParser parser;
+  public static final Set<String> FILES_TO_BE_TESTED =
```

Comment:
I don't think these have to be public.

```suggestion
class JmxMetricInsightInstallerTest {
  static final String PATH_TO_ALL_EXISTING_RULES = "src/main/resources/jmx/rules";
  private static RuleParser parser;
  static final Set<String> FILES_TO_BE_TESTED =
```